src/main/java/com/epam/ta/reportportal/core/project/settings/impl/CreateProjectSettingsHandlerImpl.java

```diff
@@ -139,17 +141,18 @@ public IssueSubTypeCreatedRS createProjectIssueSubType(String projectName, Repor
 		issueTypeRepository.save(subType);
 		projectRepository.save(project);
 
-		widgetRepository.findAllByProjectId(project.getId())
-				.stream()
-				.filter(widget -> widget.getContentFields()
-						.stream()
-						.anyMatch(s -> s.contains(subType.getIssueGroup().getTestItemIssueGroup().getValue().toLowerCase())))
-				.forEach(widget -> {
-					widget.getContentFields()
-							.add("statistics$defects$" + subType.getIssueGroup().getTestItemIssueGroup().getValue().toLowerCase() + "$"
-									+ subType.getLocator());
-					widgetRepository.save(widget);
-				});
+		String issueGroupContentField =
```

Comment:
refactor it to the separate method and with a clear name

Reply:
done